build.gradle.kts

```diff
@@ -39,7 +39,6 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        jcenter()
         maven { url = project.uri("https://maven.java.net/content/groups/public") }
         maven { url = project.uri("https://oss.sonatype.org/content/repositories/snapshots") }
```

Comment:
Are you using `senotype` to pull down the compiled `proto-kotlin` jar? If so, you don't need it because they're on MavenCentral -> https://mvnrepository.com/search?q=io.provenance

Reply:
not yet, thats on my todo list